File: smallestpos/small.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int ans=g.positive_value(arr,n);
            System.out.println(ans);
          
	}
}
class Solution
{
	public static int positive_value(int arr[],int n)
        {
        
			int smallest_value =-1;
            //Write your code here without removing the existing code
	        //the variable 'arr' contains array of integers of size n-1.
	        //modified the variable 'smallest_value' contain the output of the program in integer format.
            return smallest_value;
        }
}

File: smallestpos/small.py
# Starter Code
class Solution:
    def positive_value(self, arr, n):
       smallest_value =-1
       #Write your code here without removing the existing code
	   #the variable 'arr' contains array of integers of size n-1.
	   #modified the variable 'smallest_value' contain the output of the program in integer format.
       return smallest_value


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    ans = x.positive_value(arr, n)
    print(ans)

